docs/guides-buidler-migration.md

```diff
@@ -0,0 +1,193 @@
+---
+id: guides-buidler-migration
+title: Buidler plugin migration
+sidebar_label: Buidler plugin migration
+---
+
+##### How to migrate an existing Aragon app to the Buidler plugin
+
+This guide aims to describe the basic steps needed to migrate an Aragon app from the [aragonCLI](https://github.com/aragon/aragon-cli) to the new [Buidler plugin](https://blog.aragon.one/buidler-plugin). This new Aragon tool offers a more user-friendly and stable developer experience. You can learn more about the Buidler plugin [here](https://github.com/aragon/buidler-aragon). 
+
+For this tutorial, we will use the original [react boilerplate for aragonCLI](https://github.com/aragon/aragon-react-boilerplate/tree/react-with-cli) source code as our starting point. This guide assumes that you have a general understanding of the Aragon stack. 
+
+## 1. Install dependencies
+
+The first step is to install the following NPM development dependencies:
+
+  - **"@aragon/buidler-aragon"**: "^0.1.0"
+  - **"@aragon/contract-test-helpers"**: "0.0.1"
+  - **"@nomiclabs/buidler"**: "^1.0.2"
+  - **"@nomiclabs/buidler-truffle5"**: "^1.1.2"
+  - **"@nomiclabs/buidler-web3"**: "^1.1.2"
+  - **"bignumber.js"**: "^9.0.0"
+  - **"web3"**: "^1.2.6"
+
+Most of these dependencies should be new to your project and are related to Nomic Labs' [Buidler](https://buidler.dev), a task runner for Ethereum smart contract developers. However, one dependency that you may need to upgrade is `web3`, since Aragon's plugin requires `v1.2.6` or higher. This may introduce some breaking changes, mostly in tests. See the [Migrate tests](#5-migrate-tests) section for more information.
+
+## 2. Add/replace npm scripts
+
+Since the app won't use `aragonCLI` anymore, some NPM scripts need to be changed.
+
+In `package.json`:
+
+  - "postinstall": "npm run build-app"
+  - "build-app": "cd app && npm install && cd .."
+  - "compile": "buidler compile --force"
+  - "start": "buidler start"
+  - "test": "buidler test --network buidlerevm"
+
+Also, make sure that you have the following two scripts in `app/package.json`:
```

Comment:
Perhaps we can link to an example of what these two scripts should look like, if they're not the exact same as the previous scripts